static/app/views/admin/options.tsx

```diff
@@ -557,6 +557,24 @@ const definitions: Field[] = [
       }
     ),
   },
+  {
+    key: 'beacon.record_cpu_ram_usage',
+    label: 'RAM/CPU usage',
+    component: RadioField,
+    defaultValue: () => 'true',
+    choices: [
+      [
+        'true',
+        'Yes, I would love to help Sentry developers improve the experience of self-hosted by sending CPU/RAM usage',
+      ],
+      ['false', 'No, I would prefer to keep CPU/RAM usage private'],
+    ],
+    help: tct(
```

Comment:
Why tct here but not on the choices?

Reply:
I just copied the line above, but honestly this could use some cleaning up. We can use t() on the choices.

Reply:
done